cli/parse_pdfs.py

```diff
@@ -31,14 +31,16 @@
     PDFPageMetadata,
     StandardErrorLog,
 )
-from src.pdf_parser.pdf_utils.parsing_utils import (  # noqa: E402
-    LayoutDisambiguator,
+from src.pdf_parser.pdf_utils.disambiguate_layout import run_disambiguation_pipeline
+from src.pdf_parser.pdf_utils.ocr import (
     OCRProcessor,
-    PostProcessor,
+    extract_google_layout,
+    combine_google_lp,
 )
 
-CDN_DOMAIN = os.environ["CDN_DOMAIN"]
+sys.path.append("..")
```

Comment:
This is a bit of a smell. Why is it needed?

Reply:
not needed. old artifact from nb code. removed